src/utils/addEventListener.js

```diff
@@ -1,13 +1,13 @@
-// @flow weak
+// @flow
 
 import addEventListener from 'dom-helpers/events/on';
 import removeEventListener from 'dom-helpers/events/off';
 
-export default function(node, event, handler, options) {
-  addEventListener(node, event, handler, options);
+export default function(node: Node, event: string, handler: EventHandler, capture?: boolean) {
+  addEventListener(node, event, handler, capture);
   return {
     remove() {
-      removeEventListener(node, event, handler, options);
+      removeEventListener(node, event, handler, capture);
```

Comment:
capture vs options. I think that the specification evolved toward options in order to support passive. That can help https://github.com/oliviertassinari/react-event-listener/blob/master/src/index.js#L24.